tests/cross/src/test/scala/tests/pc/CompletionMatchSuite.scala

```diff
@@ -481,4 +481,15 @@ class CompletionMatchSuite extends BaseCompletionSuite {
                 |}""".stripMargin
     ),
   )
+  check(
+    "exhaustive-map".tag(IgnoreScala2),
+    """
+      |object A {
+      |  List(Option(1)).map{ ca@@ }
+      |}""".stripMargin,
+    """|case (exhaustive) Option (2 cases)
+       |""".stripMargin,
+    filter = _.contains("exhaustive"),
+  )
+
```

Comment:
For now I've deleted the checkEdit test, it was failing because of wrong cases ordering. I will add it back after porting sorting cases from Scala 2 in https://github.com/scalameta/metals/pull/4397

Reply:
I will bring the test back

Reply:
It's weird, the tests are failing, but they work fine locally. Here we get different ordering of completions. 
Do you have an idea what can cause that?

Reply:
Maybe it's caused by a different JDK version?